packages/design/jobberStyle.js

```diff
@@ -38,9 +38,9 @@ fs.writeFile(
       console.log("An error occured while writing SCSS to File.");
       return console.log(err);
     }
-    console.log("SCSS file has been saved.");
   },
 );
+writeMobileFoundationFiles();
```

Comment:
This function, and the related `getShadowStyles` are nicely organized... I wonder if there's a refactor we could do to some of the other "foundation" code to make the top a bit cleaner. Overall, this script is feeling a bit large and inconsistently organized.